src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -38,28 +41,46 @@ const defaultProps = {
 const ReportActionItemCreated = (props) => {
     const isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(props.report);
     const icons = ReportUtils.getIcons(props.report, props.personalDetails, props.policies);
+
+    // The image handling depends on how large the chat window is
+    const backgroundImageHeight = Dimensions.get('window').height < Dimensions.get('window').width
```

Comment:
```suggestion
    const backgroundImageHeight = (Dimensions.get('window').height < Dimensions.get('window').width)
```
NAB, I think this is more readable